ui/app/components/wizard/mounts-wizard.js

```diff
@@ -36,7 +36,7 @@ export default Component.extend({
   }),
   mountName: computed('currentMachine', 'mountSubtype', function () {
     if (this.currentMachine === 'secrets') {
-      var secret = engines().find((engine) => {
+      var secret = mountableEngines().find((engine) => {
```

Comment:
we probably want to update this to `const` right?
```suggestion
      const secret = mountableEngines().find((engine) => {
```